.gitignore

```diff
@@ -164,3 +164,5 @@ tmp-context-products.json
 
 # lidvid error log
 lidvid_error_log.txt
+
+*tmp.py
```

Comment:
What is this?

Reply:
This is telling github to ignore my local files ended with *tmp.py and don't push them to the repo. This will not affect anything unless we decide to name something *tmp.py in the future in this repo.

Reply:
I don't think the question was "WHAT is this" but "WHY is this". It seems like it could be dangerous to have an ignore rule that is so general. Maybe you could put temporary files somewhere else, or in a subdirectory that could be ignored explicitly? Also technically it's not telling it not to push, it's telling it to ignore the files when doing a "git add" or "git status".

Reply:
*tmp.py is removed from .gitignore.